packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -249,33 +249,43 @@ export const transformPlotlyJsonToDonutProps = (
   useFluentVizColorPalette: boolean,
   isDarkTheme?: boolean,
 ): IDonutChartProps => {
-  const firstData = input.data[0] as PieData;
+  const firstData = input.data[0] as Partial<PieData>;
   let colors: string[] | string | null | undefined = undefined;
   if (!useFluentVizColorPalette) {
     colors = firstData.marker?.colors ? getSchemaColors(firstData?.marker?.colors, colorMap, isDarkTheme) : undefined;
   }
 
   const mapLegendToDataPoint: Record<string, IChartDataPoint> = {};
-  firstData.labels?.forEach((label: string, index: number) => {
+  firstData.labels?.forEach((label, index: number) => {
+    let value: number;
+    if (Array.isArray(firstData.values)) {
+      if (typeof firstData.values[index] === 'number' && (firstData.values[index] as number) >= 0) {
```

Comment:
can this take negative numbers as input

Reply:
I have confirmed that Plotly ignores pie trace values if they are non-numeric or negative numbers.